apartment/services.py

```diff
@@ -87,7 +87,9 @@ def _get_haso_dynamic_part(reservation: ApartmentReservation, apartment) -> str:
 """  # noqa: E501
 
 
-def _get_price_str(cents: int) -> str:
+def _get_price_str(cents: Optional[int]) -> str:
+    if cents is None:
+        cents = 0
```

Comment:
Did you consider returning empty string for None input? Would that work for the cases where this is used?

Just asking, because it would certainly make more sense to format None as "" and 0 as "0,00 €" instead of both as "0,00 €", because they are two different things after all.

Reply:
It should be always 0,00 € because these are used in documents.

Reply:
OK